src/main/webapp/3Dscript.js

```diff
@@ -113,9 +152,9 @@ function llaDegreeToLocal(lat, lng, alt) {
    * change in our 3D space. Since altitude is already represented in
    * meters, we simply divide by 4 to adjust for our 1:4 unit ratio.
    */
-  const x = (lng - pose[0].lng) * 25000 * poseTransform.scale;
-  const y = (alt - pose[0].alt)/4;
-  const z = (lat - pose[0].lat) * 25000 * -poseTransform.scale;
+  const x = (lng - pose[5].lng) * 25000 * poseTransform.scale;
+  const y = (alt - pose[5].alt)/4;
+  const z = (lat - pose[5].lat) * 25000 * -poseTransform.scale;
```

Comment:
Could you comments why use pose at index 5?
What if there's no enough pose data?

Reply:
did this for debugging purposes and forgot to switch back